axes/attempts.py

```diff
@@ -12,17 +12,18 @@
 log = getLogger(__name__)
 
 
-def get_cool_off_threshold(attempt_time: Optional[datetime] = None) -> datetime:
+def get_cool_off_threshold(request: Optional[HttpRequest] = None) -> datetime:
```

Comment:
As the `attempt_time` is extracted from the request, I've dropped the first argument and extract it internally. 

While it's not part of the documented API, it would be a potential breaking change, if for some reasons someone uses this function directly.

Should I leave the parameter for backwards compatibility with a warning if a value is passed?